tests/materials/modified_cam_clay_test.cc

```diff
@@ -118,6 +118,26 @@ TEST_CASE("Modified cam clay undrained condition is checked in 3D",
       REQUIRE(state_variables.at("pcc") == Approx(0.0).epsilon(Tolerance));
       REQUIRE(state_variables.at("subloading_r") ==
               Approx(1.0).epsilon(Tolerance));
+
+      const std::vector<std::string> state_vars = {"bulk_modulus",

```

Comment:
Clang format issue here.
Also, not sure whether we should keep `p`, `q`, `theta` and so on as mentioned. Bulk and shear modulus might be stored as separate function too.

Reply:
Clang-format doesn't fail, so this is not an issue. 
That's for @tianchiTJ  to change?

Reply:
OK Understood. We will work with @tianchiTJ on that!